index.html

```diff
@@ -188,13 +188,14 @@ <h4 class="service-heading">Where Are We Now?</h4>
         <div class="container" style="padding-top: 100px">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h1>Contributors</h1>
+                    <h1>The Team</h1>
                     <br>
                 </div>
             </div>
             <div class="row" id="contrib">
                 <!-- contributors added here dynamically -->
             </div>
+            <h4>Other Contributors:</h4>Aarti Barai, Sriram Kovela, Bharath Kumar, Vinay Kant, Aman Bhargava, Nisha Jacob, Satya Kesav, Anirudh Deshpande, Milind Sheth, Sreetam Das, Milan Chatterjee, Chetan Rathore, Surendra Gurjar, Shubhi Saxena, Devendra Patil, Shivani Shukla, Spandan Pandey, Aroma Rodrigues.</h4>
```

Comment:
Hmm, didn't think about where exactly we'll draw the line. I considered people who contributed more than one project, and have continued to work as major contributors.

But I think we have about 30 contributors, and it doesn't make sense to have photos of all of them. Or does it?